tests/unittests/Foundation/ReferenceFoundation/TestNSURL.mm

```diff
@@ -73,25 +76,42 @@
     id testRoot =
         [NSPropertyListSerialization propertyListWithData:data options:static_cast<NSPropertyListReadOptions>(0) format:nil error:&error];
     if (!testRoot || error) {
+#ifdef _WIN32
         TraceError(L"TestNSURL", L"Unable to deserialize property list data");
+#else
```

Comment:
make a shim? (Although I don't think anyone is using these in Foundation tests, it would make it much easier, if they were to)